website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -30,23 +30,18 @@ Envoy must be run with the `--max-obj-name-len` option set to `256` or greater f
 
 ## Supported Versions
 
-Consul's Envoy support was added in version 1.3.0. The following table shows
-compatible Envoy versions.
-
-| Consul Version      | Compatible Envoy Versions                              |
-| ------------------- | ------------------------------------------------------ |
-| 1.11.x              | 1.20.2, 1.19.3, 1.18.6, 1.17.4                         |
-| 1.10.x              | 1.18.4, 1.17.4, 1.16.5, 1.15.5                         |
-| 1.9.x               | 1.16.5, 1.15.5, 1.14.7<sup>1</sup>, 1.13.7<sup>1</sup> |
-| 1.8.x               | 1.14.7, 1.13.7, 1.12.7, 1.11.2                         |
-| 1.7.x               | 1.13.7, 1.12.7, 1.11.2, 1.10.0<sup>2</sup>             |
-| 1.6.x, 1.5.3, 1.5.2 | 1.11.1, 1.10.0, 1.9.1, 1.8.0<sup>3</sup>               |
-| 1.5.1, 1.5.0        | 1.9.1, 1.8.0<sup>3</sup>                               |
-| 1.4.x, 1.3.x        | 1.9.1, 1.8.0†, 1.7.0<sup>3</sup>                       |
+The following matrix describes Envoy compatability for the currently supported **n-2 major Consul releases**. For previous Consul version compatability please view the respective versioned docs for this page. 
 
+Consul supports **four major Envoy releases** at the beginning of each major Consul release. Consul ensures that compatability with Envoy major releases are persisted across Envoy patch releases, so that fixes to major Bugs or CVEs that addressed by Envoy are supported. As a policy, only new major Envoy versions are added during the development of a upcoming Consul major release, and not to existing releases. 
```

Comment:
```suggestion
Consul supports **four major Envoy releases** at the beginning of each major Consul release. Consul maintains compatibility with Envoy patch releases for each major version so that users can benefit from bug and security fixes in Envoy. As a policy, Consul will add support for a new major versions of Envoy in a Consul major release. Support for newer versions of Envoy will not be added to existing releases.
```